integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -95,6 +95,7 @@ else
                 "k8s-shared-volume.bats" \
                 "k8s-volume.bats" \
                 "k8s-nginx-connectivity.bats" \
+				"k8s-ro-volume.bats" \
```

Comment:
Also: I think you need to remove "k8s-ro-volume.bats" from the comment above this "if" block.

Reply:
fixed